test/async-observer.test.js

```diff
@@ -194,6 +206,57 @@ describe('async observer', function() {
             done();
           });
       });
+
+    it('should allow observers to skip other ones',
+      function(done) {
+        TestModel.observe('before invoke',
+          function(context, next) {
+            notifications.push('before invoke');
+            context.callback(null, 0);
```

Comment:
Looking at this in context the name `callback` is pretty misleading since we are controlling the flow of the entire chain.